### DISCUSSION TITLE
Agenda for the Flatcar Developer Sync (Formerly Release Team Call) on Tuesday, 27th of June 9:00pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PDT

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#1079] [Flatcar Developer Sync]
- Jitsi Link: https://meet.flatcar.org/OfficeHours
- Youtube live stream / recording:  [here](https://www.youtube.com/watch?v=qwnVOQ2i7c0)
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## Agenda
1. Roadmap review (@t-lo) : https://github.com/orgs/flatcar/projects/7/views/9 
2. Projects status review:
   a. Flatcar ClusterAPI status review (@johananl ): https://github.com/orgs/flatcar/projects/7/views/14
   b. Systemd Sysext integration status review (@pothos): https://github.com/orgs/flatcar/projects/7/views/16
3. Maintainer team tactical / implementation (aka "line work") status review (@sayanchowdhury ): https://github.com/orgs/flatcar/projects/7/views/1
4. Resolved and New bugs and issues review

### Release planning
- Ongoing and upcoming releases tracker: (https://github.com/orgs/flatcar/projects/7/views/8)
- Talk about the upcoming release and the status of the packages
- Status of the ongoing release
- Preparation of the next release 

### Q&A
- Questions from community participants, answered by the Flatcar maintainers 

### Administrative tasks after meeting concluded
- [ ] Create a new Github discussion for the next Flatcar developer sync: https://github.com/flatcar/Flatcar/discussions/new?category=flatcar-developer-sync
- [ ] Close this discussion as "Resolved".

**Comment by pothos** [comment]
New issues:
- Security updates
- https://github.com/flatcar/Flatcar/issues/1096
- https://github.com/flatcar/Flatcar/issues/1094
- https://github.com/flatcar/Flatcar/issues/1093 https://github.com/flatcar/Flatcar/issues/1092 https://github.com/flatcar/Flatcar/issues/1052
- https://github.com/flatcar/Flatcar/issues/1070
- https://github.com/flatcar/Flatcar/issues/1062

Closed issues:
- https://github.com/flatcar/Flatcar/issues/1082 (Ext4 inode size)
- https://github.com/flatcar/Flatcar/issues/1077
- https://github.com/flatcar/Flatcar/issues/1080
- https://github.com/flatcar/Flatcar/issues/1059
- https://github.com/flatcar/Flatcar/issues/1051
- https://github.com/flatcar/Flatcar/issues/1048 (systemd update, got released)
- https://github.com/flatcar/Flatcar/issues/1073 https://github.com/flatcar/Flatcar/issues/769 https://github.com/flatcar/Flatcar/issues/766
- https://github.com/flatcar/Flatcar/issues/1036
- https://github.com/flatcar/Flatcar/issues/1030
